public/app/features/trails/RelatedLogs/NoRelatedLogsFoundScene.tsx

```diff
@@ -0,0 +1,29 @@
+import { SceneObjectBase, type SceneObjectState } from '@grafana/scenes';
+import { Stack, Text, TextLink } from '@grafana/ui';
+import { Trans } from 'app/core/internationalization';
+
+export class NoRelatedLogsScene extends SceneObjectBase<SceneObjectState> {
+  static readonly Component = () => {
+    return (
+      <Stack direction="column" gap={1}>
+        <Text color="warning">
+          <Trans i18nKey="explore-metrics.related-logs.warnExperimentalFeature">
+            Related logs is an experimental feature.
+          </Trans>
+        </Text>
+        <Text>
+          <Trans i18nKey="explore-metrics.related-logs.relatedLogsUnavailableBeforeDocsLink">
+            Related logs are not available for this metric. Try selecting a metric created by a{' '}
+          </Trans>
+          <TextLink external href="https://grafana.com/docs/loki/latest/alert/#recording-rules">
+            <Trans i18nKey="explore-metrics.related-logs.docsLink">Loki Recording Rule</Trans>
+          </TextLink>
+          <Trans i18nKey="explore-metrics.related-logs.relatedLogsUnavailableAfterDocsLink">
+            , or check back later as we expand the various methods for establishing connections between metrics and
+            logs.
+          </Trans>
```

Comment:
TIL @darrenjaneczek told me if there is no scene related thing in the component then we don't need to use a scene component. It can be a pure react component.

Reply:
The way we're using this currently is:

https://github.com/grafana/grafana/blob/a0a9d577ae2ac1e24efceec66d6b532f6c105849/public/app/features/trails/RelatedLogs/RelatedLogsScene.tsx#L69-L71

The type of `body` is `SceneObject<SceneObjectState> | undefined`, so I'm not sure how to use a plain React component here. @darrenjaneczek do you have an example handy of how to accomplish that?